README.md

```diff
@@ -203,6 +203,15 @@ fabric --help
 
 ### Using the `fabric` client
 
+If you want to use it with OpenAI API compatible inference servers, such as [FastChat](https://github.com/lm-sys/FastChat) or [Helmholtz Blablador](http://helmholtz-blablador.fz-juelich.de), simply export the following environment variables:
+
+- `export OPENAI_BASE_URL=https://YOUR-SERVER:8000/v1/`
```

Comment:
Change the environment variable to `OPENAI_API_BASE_URL` which I've seen used elsewhere and is used in #334 

Also add "LM Studio" at https://lmstudio.ai/ to the list of OpenAI compatible servers.

Reply:
Hi Kayvan, 

The variable is `OPENAI_BASE_URL`, without the `API` : 
https://github.com/openai/openai-python/blob/0cbce3f6777fd8f137ed7445e9bc2f2aa5c9f5b4/src/openai/_client.py#L110

As you can see, there is no mention to this name on their code at all: 
https://github.com/search?q=repo%3Aopenai%2Fopenai-python%20OPENAI_API_BASE_URL&type=code

Reply:
Awesome. My PR is unnecessary, then. I'll test and close it. Thank you.